proposals/system-test/sample-apps/jaxrs-mpjwt/src/test/java/org/eclipse/microprofile/system/test/app/SecuredSvcTest.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright (c) 2019 IBM Corporation and others
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.eclipse.microprofile.system.test.app;
+
+import static org.junit.Assert.assertTrue;
+
+import javax.inject.Inject;
+
+import org.eclipse.microprofile.system.test.jupiter.JwtBuilder;
+import org.eclipse.microprofile.system.test.jupiter.JwtConfig;
+import org.eclipse.microprofile.system.test.jupiter.MicroProfileTest;
+import org.junit.jupiter.api.Test;
+import org.testcontainers.containers.microprofile.MicroProfileApplication;
+import org.testcontainers.junit.jupiter.Container;
+
+@MicroProfileTest
+public class SecuredSvcTest {
+    
+    @Container
+    public static MicroProfileApplication<?> app = new MicroProfileApplication<>()
+                    .withAppContextRoot("/myservice")
+                    .withEnv("mp_jwt_verify_publickey", JwtBuilder.getPublicKey())                   
```

Comment:
Eventually I would like the test framework to automatically generate/set the `mp_jwt_verify_publickey` and `mp_jwt_verify_issuer` properties on the container if `@JwtConfig` usage is detected anywhere else in the class, but this can be done in a follow-on PR